test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -383,6 +383,27 @@ class CharacterTest {
     assertFalse(Character.isDigit('\uFBFC'))
   }
 
+  @Test def codePointCount(): Unit = {
+    assertEquals(1, Character.codePointCount("\uD800\uDC00", 0, 2))
+    assertEquals(1, Character.codePointCount("\uD800\uDC01", 0, 2))
+    assertEquals(1, Character.codePointCount("\uD801\uDC01", 0, 2))
+    assertEquals(1, Character.codePointCount("\uDBFF\uDFFF", 0, 2))
+    assertEquals(3, Character.codePointCount("a\uD800\uDC00b", 0, 4))
+    assertEquals(4, Character.codePointCount("a\uD800\uDC00b\uD800", 0, 5))
+
+    // "No NPE, null char sequence."
+    assertThrows(classOf[NullPointerException], Character.codePointCount(null.asInstanceOf[CharSequence], 0, 1))
```

Comment:
As I said last time, we normally don't test NPEs, because they're Undefined Behavior.